drivers/gpio/gpio_brcmstb.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2024 Junho Lee <junho@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT brcm_brcmstb_gpio
+
+#include <zephyr/arch/common/sys_bitops.h>
+#include <zephyr/arch/cpu.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/gpio/gpio_utils.h>
+
+#define GIO_DATA  0x04
+#define GIO_IODIR 0x08
+
+#define DEV_CFG(dev)  ((const struct gpio_brcmstb_config *)(dev)->config)
+#define DEV_DATA(dev) ((struct gpio_brcmstb_data *)(dev)->data)
+
+struct gpio_brcmstb_config {
+	struct gpio_driver_config common;
+
+	DEVICE_MMIO_NAMED_ROM(reg_base);
+	mem_addr_t offset;
+};
+
+struct gpio_brcmstb_data {
+	struct gpio_driver_data common;
+
+	DEVICE_MMIO_NAMED_RAM(reg_base);
+	mem_addr_t base;
+
+	sys_slist_t cb;
```

Comment:
Is it used?

Reply:
No, it isn't used right now. It seems I forgot to remove it. Thank you.